Dockerfile.centos

```diff
@@ -0,0 +1,49 @@
+FROM centos:7
+
+LABEL maintainer="yee.yi@vesoft.com"
+
+RUN yum update -y \
+  && yum install -y centos-release-scl \
+  && yum install -y devtoolset-8 \
+  && yum install -y \
+    autoconf \
+    autoconf-archive \
+    automake \
+    file \
+    git \
+    java-1.8.0-openjdk \
+    libtool \
+    make \
+    maven \
+    ncurses-devel \
+    perl \
+    perl-WWW-Curl \
+    readline-devel \
+    rpm-build \
+    unzip \
+    vim \
+    wget \
+    xz-devel \
```

Comment:
install gdb

Reply:
Does `curl` is needed ？

Reply:
`gdb` has been installed by `devtoolset-8` package and `curl` also has been installed by other dependencies.